dev-python/sortedcontainers/sortedcontainers-2.1.0.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Python library to sort collections and containers"
+HOMEPAGE="http://www.grantjenks.com/docs/sortedcontainers/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND=""
+DEPEND="${RDEPEND}
+	test? ( dev-python/pytest[${PYTHON_USEDEP}] )"
+
+python_test() {
```

Comment:
This can be dropped in favor of calling distutils_enable_tests pytest in global scope. See distutils-r1.eclass for an example

Reply:
Fixed in new pull request: https://github.com/gentoo/gentoo/pull/14014